napari/_qt/dialogs/_tests/test_reader_dialog.py

```diff
@@ -149,20 +150,20 @@ def _(path):
         uri='some-path/some-file.fake',
     )
     tmp_plugin.manifest.contributions.sample_data = [my_sample]
-    from napari._qt.dialogs.qt_reader_dialog import QtReaderDialog
 
+    app = get_app()
+    # required so setup steps run in init of `Viewer` and `Window`
     viewer = make_napari_viewer()
-    sample_action = viewer.window.file_menu.open_sample_menu.actions()[0]
-
+    # Ensure that `handle_gui_reading` called with `plugin_name=None`
```

Comment:
nit - technically `plugin_name` does not appear in that function call below, as it's using the default arg. So maybe just update this comment to say `Ensure that handle_gui_reading is not passed the sample plugin name` or something like that

Reply:
Also maybe worth linking to the PR that fixed this